### DISCUSSION TITLE
Prevent Neorg from changing CWD when using GTD

### DISCUSSION THREAD

**Discussion by vanhtuan0409** [nvim-neorg/neorg#224] [Q&A] [answered]
Hi, is there any way that we can prevent Neorg to change root CWD when using GTD?

For example, I want to take a quick look over my todo list using `Neorg gtd views`. This cause the pwd change to Neorg workspace and prevent me to continue working on the project.

My investigation is that some `BufEnter` autocmd was triggered and causing Neorg to switch the cwd

Step to reproduce:
1. Open Nvim at `~/foo`
2. Run `pwd` in command mode => result should be `~/foo`
3. Start Neorg via `NeorgStart silent=true`
4. Open GTD view by `Neorg gtd views`
5. Check `pwd` again in command mode => expected to be `~/foo` but now changed to GTD workspace instead

**Comment by max397574** [comment]
`NeorgStart silent=true`

**Comment by danymat** [comment]
Hello, I saw that you're the same person that asked it in https://github.com/nvim-neorg/neorg/discussions/223 , did you try it out ?

**Reply by vanhtuan0409** [reply]
Yeah I tried it out. This is actually a different question. `NeorgStart silent=true` do start Neorg. Said you can follow these steps:

1. Open Nvim at `~/foo`
2. Run `pwd` in command mode => result should be `~/foo`
2. Start Neorg via `NeorgStart silent=true`
3. Open GTD view by `Neorg gtd views`
4. Check `pwd` again in command mode => expected to be `~/foo` but now changed to GTD workspace instead

**Comment by danymat** [comment, marked as answer]
Hello, this should be implemented in https://github.com/nvim-neorg/neorg/pull/275
